mvrx/src/main/kotlin/com/airbnb/mvrx/DeliveryMode.kt

```diff
@@ -7,6 +7,8 @@ import kotlin.reflect.KProperty1
  * See: [RedeliverOnStart], [UniqueOnly].
  */
 sealed class DeliveryMode {
+    abstract val subscriptionId: String
```

Comment:
as I see you still use `subscriptionId` only for UniqueOnly mode, do you need this change?

Reply:
technically not with this current implementation. but i do feel it makes it does make senses for the different delivery modes to provide one.